hibernate-reactive-core/src/test/java/org/hibernate/reactive/MutinySessionTest.java

```diff
@@ -56,6 +56,32 @@ private static String nameFromResult(List<Object> rowSet) {
 		}
 	}
 
+	@Test
+	public void testClearSessionAndChangeName(TestContext context) {
+		final GuineaPig pig1 = new GuineaPig( 81, "pig1" );
+		final GuineaPig pig2 = new GuineaPig( 82, "pig2" );
+
+		test( context,
+				getMutinySessionFactory().withSession( session -> session
+						.persistAll( pig1, pig2 )
+						.invoke( session::clear )
+						.invoke( session::flush )
```

Comment:
Not sure if it's intentional, but this is different from what the user was doing. The user first flush and then clear.

Reply:
I've just noticed that flush should be  `.call(session::flush)`